src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer_IncrementalAnalyzer.cs

```diff
@@ -22,7 +22,7 @@ private partial class DiagnosticIncrementalAnalyzer
         /// <summary>
         /// Cached data from a real <see cref="Project"/> instance to the cached diagnostic data produced by
         /// <em>all</em> the analyzers for the project.  This data can then be used by <see
-        /// cref="DiagnosticGetter.ProduceDiagnosticsAsync"/> to speed up subsequent calls through the normal <see
+        /// cref="GetDiagnosticsForIdsAsync"/> to speed up subsequent calls through the normal <see
```

Comment:
>GetDiagnosticsForIdsAsync [](http://example.com/codeflow?start=18&length=25)

Should we consider making these new CWTs static to avoid any possibility of the issue that's been my bane today?

Reply:
Will do!